index.js

```diff
@@ -43,6 +43,7 @@ class CountDown extends React.Component {
     until: Math.max(this.props.until, 0),
     lastUntil: null,
     wentBackgroundAt: null,
+    appStateSubscription: null
```

Comment:
@Lamonarch87 
If you use TypeScript in your project, assign this **PropType** to **appStateSubscription**, to avoid warnings when rendering the component:
`appStateSubscription: PropTypes.oneOfType([PropTypes.func, PropTypes.object])`